tests/unit/PaymentErrorDialog.spec.ts

```diff
@@ -138,4 +138,81 @@ describe('Payment Error Dialog', () => {
 
     wrapper.destroy()
   })
+
+  it('renders PAD error messages correctly when they are present', () => {
+    store.state.keycloakRoles = []
+
+    const wrapper = shallowMount(PaymentErrorDialog,
+      {
+        propsData: {
+          dialog: true,
+          errors: [{ message: 'error msg' }]
+        },
+        store,
+        vuetify
+      })
+    const vm: any = wrapper.vm
+
+    expect(vm.numErrors).toBe(1)
+    expect(wrapper.find('#dialog-title').text()).toBe('Unable to Process Payment')
+    expect(wrapper.find('#dialog-text').text()).toContain('This Filing has been')
+    expect(wrapper.find('#dialog-text').text())
+      .toContain('We were unable to process your payment due to the following errors:')
+    expect(wrapper.find('#dialog-text').text()).toContain('error msg')
+    expect(wrapper.find('#dialog-ok-button')).toBeDefined()
+    
+    expect(wrapper.isVisible()).toBe(true)
+
+    expect(wrapper.findAll('p').length).toBe(3)
+    expect(wrapper.findAll('p').at(0).text()).toContain('We are unable to process your payment')
+    expect(wrapper.findAll('p').at(1).text()).toContain(
+      'We were unable to process your payment due to the following errors:'
+    )
+    expect(wrapper.findAll('p').at(2).text()).toContain('If this error persists')
+    expect(wrapper.findAll('li').length).toBe(0)
+    expect(wrapper.findAll('span').length).toBe(2)
+    expect(wrapper.findAll('span').at(1).text()).toContain('error msg')
+
+    // expect(wrapper.findComponent(ErrorContact).exists()).toBe(true)
```

Comment:
is there a reason why we are leaving this here?  same with test below.

Reply:
Thanks. I forgot to remove it.